releasenotes/notes/apm-start-up-error-logging-3e927cdbb059db57.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    APM : Ensured start-up errors are logged to the log file
```

Comment:
Nit:
There should be no space between "APM" and ":"

Reply:
I was lucky enough to open releasenotes(to check it out as an example) that do have a space between these two. I'll delete that then